internal/command/meta_config.go

```diff
@@ -150,6 +159,21 @@ func (m *Meta) rootModuleCall(rootDir string) (configs.StaticModuleCall, tfdiags
 	return call, diags
 }
 
+func (m *Meta) getInput(ctx context.Context, variable *configs.Variable) (string, error) {
```

Comment:
What happens when this function is run in automation?

https://opentofu.org/docs/cli/config/environment-variables/#tf_input